File: primerTrimestre/introduccion_programacion_python/1secuenciales_py/ej3sec.py
# Programa: ej3sec.py
# Propósito: Dados los catetos de un triángulo rectángulo, calcular su hipotenusa.
# Autor: Antonio Quesada
# Fecha: 13/10/2019.
#
# Variables a usar:
#   * cateto1
#   * cateto2
#   * hipotenusa: hipotenusa^2 = cateto1^2+cateto2^2
#
# Algoritmo:
#   LEER cateto1, cateto2
#   hipotenusa^2 <-- cateto1^2+cateto2^2
#   ESCRIBIR hipotenusa

import math
#from math import sqrt

# Petición de datos
cateto1 = float(input("Introduce el cateto 1: "))
cateto2 = float(input("Introduce el cateto 2: "))
print("-------------------------------------------")

#Cálculos. Teorema de pitágoras.
hipotenusa = round(math.sqrt(cateto1**2+cateto2**2),4)

# Salida
print("La hipotenusa del triángulo es", hipotenusa)

File: primerTrimestre/java basico/1secuenciales/Ej3sec.java
/**Dados los catetos de un triángulo rectángulo, calcular su hipotenusa.
 * 
 * @author Antonio Quesada
 */

import java.util.Scanner;
import java.math.*;

public class Ej3sec {
  public static void main(String[] args) {
    Scanner s = new Scanner(System.in);
    
    //Definimos las variables.
    
    double cateto1;
    double cateto2;
    double hipotenusa;
    
    //Pedimos los catetos y lo leemos.
    System.out.println("Introduce el primer cateto:");
    cateto1 = s.nextDouble();
    System.out.println("Introduce el segundo cateto:");
    cateto2 = s.nextDouble();
    
    //Calculamos la hipotenusa por el teorema de Pitágoras importando math.
    hipotenusa = Math.sqrt(Math.pow(cateto1,2)+Math.pow(cateto2,2));
    
    //Mostramos el resultado.
    System.out.println("La hipotenusa es "+hipotenusa);


  }   
}
